lib/gitlib.sh

```diff
@@ -189,7 +191,7 @@ gitlib::current_branch () {
 
   [[ -n "$repo_dir" ]] && git_args=("-C" "$repo_dir")
 
-  if ! git ${git_args[*]} rev-parse --abbrev-ref HEAD 2>/dev/null; then
+  if ! git "${git_args[*]}" rev-parse --abbrev-ref HEAD 2>/dev/null; then
```

Comment:
We want each element of the `git_args` array to be treated as its own shell word; `[*]` will expand to only one word
```suggestion
  if ! git "${git_args[@]}" rev-parse --abbrev-ref HEAD 2>/dev/null; then
```

Reply:
Thanks a lot for your review. I made the change